src/main/kotlin/gg/essential/multi-version.api-validation.gradle.kts

```diff
@@ -13,24 +15,26 @@ val mergedApiFile = file("api/$name.api")
 val apiDump by tasks.registering(MergeApiFiles::class) {
     output.set(mergedApiFile)
     evaluationDependsOnChildren()
-    inputs.from(subprojects.map { project ->
+    inputs.from(multiversionChildProjects.map { (_, project) ->
         project.tasks.named("apiDump").map { project.file("api/${project.name}.api") }
     })
 }
 
 subprojects {
-    val projectApiDir = project.file("api").also { it.mkdirs() }
-    val extractApiDefinition by tasks.registering(ExtractApiFile::class) {
-        selector.set(project.name)
-        input.set(mergedApiFile)
-        output.set(projectApiDir.resolve("${project.name}.api"))
-    }
-    afterEvaluate {
-        tasks.named("apiCheck") {
-            dependsOn(extractApiDefinition)
+    if (isMultiversionChildProject) {
```

Comment:
Not sure this is guaranteed to work. If this plugin is applied before the child project is configured, then the multiversion plugin won't yet be applied to the child project and so this might return `false`?
Should probably use [`withPlugin`](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api.plugins/-plugin-manager/with-plugin.html) instead.